src/components/singular-product/singular-product.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
```

Comment:
This looks great so far!  Can you add this to the product template page in "/templates/product.js" and ultimately remove the singular product page javascript file as it is only necessary for generated product pages.